model/build.gradle

```diff
@@ -1,3 +1,7 @@
 java {
   sourceCompatibility = JavaVersion.VERSION_1_8
 }
+
+dependencies {
+  implementation 'org.gradle:gradle-tooling-api:8.8'
+}
```

Comment:
Is this needed now `@Nullable` is gone?

Reply:
Yes it's not needed. I'm just about to append a commit to remove it